Tests/TestCodeGenConfigurations/EmbeddedInTarget-InSchemaModule/Tests/MySwiftPackageTests/TestMocks/Bird+Mock.swift

```diff
@@ -1,54 +0,0 @@
-// @generated
```

Comment:
These meant to be deleted?

Reply:
Yeah, meant to delete them in the last PR, but its alright. haha